run-system-tests/action.yml

```diff
@@ -155,18 +156,19 @@ runs:
           go get github.com/0chain/gosdk@${{ inputs.custom_go_sdk_version }} || { echo "::error title=Failed to retrieve gosdk::failed to retrieve gosdk [${{ inputs.custom_go_sdk_version }}]" && exit 1; }
           go mod tidy || { echo "::error title=Failed to upgrade gosdk on zwallet CLI::failed to upgrade zwallet CLI gosdk to [${{ inputs.custom_go_sdk_version }}]" && exit 1; }
         fi
-        make install > build.log 2>&1 || { cat build.log && echo "::error title=zwallet CLI build failed::zwallet CLI build failed" && exit 1; }
-        mv zwallet ../tests/cli_tests
-        echo "Building zboxcli [${{ env.ZBOX_BRANCH }}]..."
+        make install > build.log 2>&1 || { cat build.log && echo "::error title=zwallet CLI build failed::zwallet CLI build failed" && exit 1; } && \
```

Comment:
if the build fails eg. exit 1 is called how will this act if the process is running in the background?

Reply:
I misread the docs, pushed a fix